### DISCUSSION TITLE
Send raw

### DISCUSSION THREAD

**Discussion by Jeradam** [zwave-js/zwave-js-ui#3667] [Q&A]
Hi,

I would like to separate input and output on a fgbs 222, I have find the solution here :
https://community.homey.app/t/seperate-the-input-from-output-fibaro-smart-implant/28950/31

But is it possible (and how) to "send raw" with zwave-js ?
Thanks!

**Comment by kpine** [comment]
Why do you need to send raw commands? Protection CC is supported by Z-Wave JS.

**Reply by Jeradam** [reply]
Because I didn't see this parameter ☺️
Thanks!
